db_stress_tool/db_stress_common.cc

```diff
@@ -321,6 +321,16 @@ uint32_t GetValueBase(Slice s) {
   return res;
 }
 
+AttributeGroups GenerateAttributeGroups(
+    const std::vector<ColumnFamilyHandle*>& cfhs, uint32_t value_base,
+    const Slice& slice) {
+  AttributeGroups attribute_groups;
+  for (auto* cfh : cfhs) {
+    attribute_groups.emplace_back(cfh, GenerateWideColumns(value_base, slice));
```

Comment:
If we're going to write the same thing into each CF, we can just call `GenerateWideColumns` once, right?